main.go

```diff
@@ -2,9 +2,12 @@ package main
 
 import (
 	"fmt"
+
+	log "github.com/sirupsen/logrus"
```

Comment:
:thinking: Do we need a third-party library for this?

Reply:
We don't need these changes at all.  I'll remove the file from the PR.  Good catch.